data/ke2022daily/ke2022daily-extraction.md

```diff
@@ -0,0 +1,111 @@
+                                             value=Ke2022.loc[j,"saliva_conc"]) for j in Ke2022.loc[(Ke2022["Ind"]==i) & (pd.notna(Ke2022['saliva_conc']))].index]) for i in pd.unique(Ke2022["Ind"])]
+
+ke2022 = dict(title="Daily Longitudinal Sampling of SARS-CoV-2 Infection Reveals Substantial Heterogeneity in Infectiousness",
+               doi="10.1038/s41564-022-01105-z",
+               description=folded_str("The authors studied the dynamics of infectious virus and viral RNA shedding for SARS-CoV-2 during acute infection through daily longitudinal sampling of 60 individuals for up to 14 days. Nasal swab and saliva samples were collected daily and tested for University of Illinois at Urbana-Champaign faculty, staff, and students (during the fall of 2020 and spring of 2021) who reported a negative RT-qPCR test result in the past 7 days and were either within 24 h of a positive RT-qPCR result or within 5 days of exposure to someone with a confirmed positive RT-qPCR result.\n"),
+               analytes=dict(nasal_SARSCoV2=dict(description=folded_str("SARS-CoV-2 RNA genome copy concentration in mid-turbinate nasal swab (nasopharyngeal swab) samples. Note that the unit of these measurements is per mL: this is because nasal swab samples were each collected in 3 mL of VTM. The calibration curve for nasal samples was in the Supplemental Table S10 in Ke et al. (2022).\n"),
+                                                    specimen="nasopharyngeal_swab",
+                                                    biomarker="SARS-CoV-2",
+                                                    gene_target="N1 and N2",
+                                                    limit_of_quantification=0.22387211385683378, #calculated by 10**(11.35-0.25*48) with CN=48; the minimum quantifiable value observed was 7.89768849399884;
+                                                    limit_of_detection="unknown",
+                                                    unit="gc/mL",
+                                                    reference_event="confirmation date"),
+                             saliva_SARSCoV2=dict(description=folded_str("SARS-CoV-2 RNA genome copy concentration in saliva samples. The study was not able to measure the calibration curve using saliva samples taken from participants. Instead, the authors used data from calibration experiments in which saliva samples obtained from healthy donors were spiked with SARS-CoV-2 genomic RNA. The calibration curve for saliva samples was in the Supplemental Table S11 in Ke et al. (2022).\n"),
+                                              specimen="saliva",
+                                              biomarker="SARS-CoV-2",
+                                              gene_target="N",
+                                              limit_of_quantification=12.022644346174081, #calculated by 10**(14.24-0.28*47) with CT=47; the minimum quantifiable value observed was 1161.9836038697981
```

Comment:
Do you have a hunch for if the value based on CN=48 best represents LOD or LOQ? Just wondering because the limit is almost two orders of magnitude lower than the smallest observed value.

Reply:
I think those values are quite conservative and I am not sure whether they are LOD of LOQ. Do you think we should change those back to `unknown`?

Reply:
Maybe that is the best approach given that we don't know for sure. Maybe we can add the Ct value and estimated viral load at that Ct value to the description and leave the formal values as `unknown`?

Reply:
I changed it back to `unknown` and left some information in the comments.